### PR TITLE
Add Rancher Desktop slider component

### LINKED ISSUE #2464: Update slider style to remove borders and match mockups

### PR DESCRIPTION
**Pull request by rak-phillip**
This adds a rancher desktop-specific slider component so that we can more readily style our sliders without having to build each slider so that each instance has a matching template. 

closes #2464 

Signed-off-by: Phillip Rak <rak.phillip@gmail.com>

### CODE REVIEW

File: src/components/form/RdSlider.vue [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  .vue-slider::v-deep .vue-slider-dot-handle {
+    background-color: var(--checkbox-tick-disabled);
+  }
+}
+
+.vue-slider::v-deep .vue-slider-process {
+  background-color: var(--error);
+}
+
+.slider-container {
```
Comment (ericpromislow) [inline src/components/form/RdSlider.vue:126]: This class isn't used anywhere?

File: src/components/__tests__/SystemPreferences.spec.js [hunk 2]
```diff
@@ -64,22 +64,22 @@ describe('SystemPreferences.vue', () => {
```
Comment (ericpromislow) [inline src/components/__tests__/SystemPreferences.spec.js:64]: A couple of lines before this there's an existing reference to `noChangesToApply` in `delete minimalProps.noChangesToApply` but this field doesn't appear anywhere else.

**Review by ericpromislow — CHANGES_REQUESTED** [review]
Looks like some unused code -- fine otherwise

File: src/components/form/RdSlider.vue (hunk 1 above)
Comment (rak-phillip) [reply src/components/form/RdSlider.vue]: Good eye, this looks like an old class that survived a refactoring 😁

File: src/components/__tests__/SystemPreferences.spec.js (hunk 2 above)
Comment (rak-phillip) [reply src/components/__tests__/SystemPreferences.spec.js:63]: This must have been leftover from a much older implementation.. removed.

**Review by ericpromislow — APPROVED** [review]
LG

**Pull request merged**